src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke.tasks;
+
+import java.time.LocalDateTime;
+
+import duke.Time;
+
+/**
+ * @author Damith C. Rajapakse
+ * Reused from https://nus-cs2103-ay2021s1.github.io/website/schedule/week2/project.html with modifications
+ *
+ * Class to initiate a deadline task.
+ */
+public class Deadline extends Task {
+    private LocalDateTime by;
+
+    /**
+     * Constructor for Deadline.
+     */
+    public Deadline(String description, LocalDateTime by) {
```

Comment:
Consider renaming the field by for easier understanding

Reply:
Alright! Will make the change.